src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/ttd/presentationfields-app/logic/Calculation/CalculationHandler.cs

```diff
@@ -20,8 +22,10 @@ public static class CalculationHandler
         /// }
         /// </example>
         /// <param name="instance">The data</param>
-        public static async Task<bool> Calculate(object instance)
+        /// <param name="altinnAppContext">The context</param>
+        public static async Task<bool> Calculate(object instance, IAltinnAppContext altinnAppContext)
         {
+            AltinnAppContext appContext = altinnAppContext.GetContext();
```

Comment:
can't see that the context is actuall used here? making the calculation result dependent on something in the contet would help to verify that the context is populated as expected I guess

Reply:
I added a calculation using partyId. Just to very there is a int there